### PR TITLE
[Maintenance] Adding Aria labels for accessibility

### PR DESCRIPTION
**Pull request by MikesGlitch**
Adding aria-labels to some buttons to improve accessibility score. 

Notes: 
- There's still loads of issues left - I've only tackled a few here
- Color contrast is a common issue across pages
- Might want to start adding data-test-id to elements to make the Puppeteer tests more resilient

### CODE REVIEW

File: packages/desktop-client/src/components/Titlebar.tsx [hunk 1]
```diff
@@ -114,6 +114,7 @@ function PrivacyButton({ style }) {
   return (
     <Button
       type="bare"
+      aria-label="Toggle Privacy"
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/Titlebar.tsx:117]: **🥜 nitpick:** ‏would be better to be explicit here for visually impaired folks.

```suggestion
      aria-label={`${isPrivacyEnabled ? 'Disable' : 'Enable'} privacy mode`}
```

File: packages/desktop-client/src/components/accounts/Header.js [hunk 2]
```diff
@@ -168,6 +168,7 @@ export function AccountHeader({
                 )}
                 <Button
                   type="bare"
+                  aria-label="Edit"
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/accounts/Header.js:171]: **🥜 nitpick:** ‏
```suggestion
                  aria-label="Edit account name"
```

File: packages/desktop-client/src/components/budget/report/budgetsummary/BudgetSummary.tsx [hunk 3]
```diff
@@ -84,6 +84,7 @@ export default function BudgetSummary({ month }: BudgetSummaryProps) {
           >
             <Button
               type="bare"
+              aria-label="Toggle Summary"
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/budget/report/budgetsummary/BudgetSummary.tsx:87]: **🥜 nitpick:** ‏would be good to be explicit here too

File: packages/desktop-client/src/components/budget/rollover/budgetsummary/BudgetSummary.tsx [hunk 4]
```diff
@@ -86,6 +86,7 @@ export default function BudgetSummary({
           >
             <Button
               type="bare"
+              aria-label="Toggle Summary"
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/budget/rollover/budgetsummary/BudgetSummary.tsx:89]: and here

File: packages/desktop-client/src/components/sidebar/ToggleButton.tsx [hunk 5]
```diff
@@ -15,7 +15,12 @@ type ToggleButtonProps = {
 function ToggleButton({ style, isFloating, onFloat }: ToggleButtonProps) {
   return (
     <View className="float" style={{ ...style, flexShrink: 0 }}>
-      <Button type="bare" onClick={onFloat} color={theme.buttonMenuBorder}>
+      <Button
+        type="bare"
+        aria-label="Toggle Sidebar"
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/sidebar/ToggleButton.tsx:20]: and here

File: packages/desktop-client/src/components/util/AmountInput.tsx [hunk 6]
```diff
@@ -88,6 +88,7 @@ export function AmountInput({
       leftContent={
         <Button
           type="bare"
+          aria-label="Toggle Positive/Negative"
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/util/AmountInput.tsx:91]: and here

**Review by MatissJanis — COMMENTED** [review]
Overall LGTM! Let me know what you think of my comments.

Happy to merge even without them being resolved as this is definitely an improvement on what we have now! :)

**Comment by MikesGlitch** [comment]
> Overall LGTM! Let me know what you think of my comments.
> 
> Happy to merge even without them being resolved as this is definitely an improvement on what we have now! :)

Sweet - done. I also made them all "Sentence case" for consistency

**Review by MatissJanis — APPROVED** [review]

**Pull request merged**